src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+type Props = {
+  user: User
+};
+
+export const UserInfo:React.FC<Props> = ({ user }) => {
+  const { name, email } = user;
```

Comment:
Consider destructuring the `user` object in the function argument instead of inside the function body. This will make the code more concise and readable. 🤓